third_party/VENDOR-LICENSE

```diff
@@ -266,214 +266,6 @@ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 
 
-===========================================================
-Import: github.com/knative/eventing-sources/vendor/github.com/aws/aws-sdk-go
```

Comment:
Hmm, I'm surprised this went away... we should make sure it ends up in the docker image for the source controller.

(Since that is how we are ensuring that a copy of the licenses are distributed with our binaries.)

Reply:
The path got removed from hack/update-deps.sh in one rebase. Restored (the license entry should come back.)